lib/wor/batchifier.rb

```diff
@@ -17,10 +23,14 @@ def execute_in_batches(collection, batch_size: 100, strategy: :add)
     protected
 
     def merge(response, rec, strategy)
-      return rec.merge(response) { |_, v1, v2| v1 + v2 } if strategy == :add
-      return rec.merge(response) { |_, v1, _| v1 } if strategy == :maintain_unique
-      return {} if strategy == :no_response
-      raise WrongStrategy
+      return Wor::Batchifier.classify_strategy(strategy).new.merge_strategy(response,rec)
+    end
+
+    def classify_strategy(strategy)
+      strategy_class_name = strategy.to_s.split('_').collect(&:capitalize).join
```

Comment:
i think that `stategy.to_s.classify` does the same here

Reply:
It's pure ruby so it doesn't have the method classify which is from rails